docs/content/en/3.api/2.$img.md

```diff
@@ -1,11 +1,11 @@
 ---
 menuTitle: $img
 title: $img
-description: 'Using $img utility to generate provider specific URLs'
+description: 'Using $img programmatically'
```

Comment:
The purpose is exactly generating URLs :D (not necessarily transform) and usage is practically possible within a `programmatic` context (anywhere in source code)